templates/common/infra/bicep/core/host/container-apps-auth.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Container Apps Auth Config using Microsoft Entra as Identity Provider.'
```

Comment:
Please add more comments description that adds more detail about when a customer would would to use this module? What use case does it solve?

Reply:
Would a link to learn.ms docs work here (e.g. https://learn.microsoft.com/en-us/azure/container-apps/authentication) ? I was trying to fit the style of other resources in this repo in terms of metadata/descriptions/... Not sure what level of detail you want